### ISSUE TITLE
lxc_container module error ValueError: too many values to unpack

### ISSUE THREAD

**Issue by dkhabarov** [ansible/ansible-modules-extras#1170]
Ansible version: 1.9.2

``` shell
failed: [server] => (item={'network_ipv4_address': '10.0.3.2', 'network_ipv4_broadcast': '10.0.3.255', 'state': 'started', 'name': 'webrails2', 'network_ipv4_gateway': '10.0.3.1'}) => {"failed": true, "item": {"name": "webrails2", "network_ipv4_address": "10.0.3.2", "network_ipv4_broadcast": "10.0.3.255", "network_ipv4_gateway": "10.0.3.1", "state": "started"}, "parsed": false}
OpenSSH_6.6.1, OpenSSL 1.0.1f 6 Jan 2014
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: Applying options for *
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Traceback (most recent call last):
  File "<stdin>", line 3084, in <module>
  File "<stdin>", line 1474, in main
  File "<stdin>", line 1371, in run
  File "<stdin>", line 971, in _started
  File "<stdin>", line 964, in _started
  File "<stdin>", line 622, in _config
ValueError: too many values to unpack
```

Task:

``` yaml
- name: manage containers
  lxc_container:
    name: "{{ item.name }}"
    template: "{{ ansible_distribution | lower }}"
    state: "{{ item.state | default('started') }}"
    container_config:
      - "lxc.start.auto = 1"
      - "lxc.network.ipv4 = {{ item.network_ipv4_address }}"
      - "lxc.network.ipv4.gateway = {{ item.network_ipv4_gateway }}"
      - "lxc.mount.entry = /data data/ none rw,bind,uid=33,gid=33 0 0"
    template_options: "--release {{ ansible_distribution_release }}"
  with_items: containers
  when: is_container_host is defined and item.state == 'started'
  tags: configure_lxc
```

Vars:

``` yaml
containers:
  - {name: 'webrails1', network_ipv4_gateway: 10.0.3.1, network_ipv4_address: 10.0.3.2, network_ipv4_broadcast: 10.0.3.255, state: 'started' }
```

OS: Ubuntu 14.4

**Comment by machacekondra** [comment]
I think this[1] should fix your issue. So you should update.

[1] https://github.com/ansible/ansible-modules-extras/pull/639

**Comment by dkhabarov** [comment]
Thx.

**Comment by dkhabarov** [comment]
Fix #639 does not solve my problem. The error remained.

**Comment by mightydok** [comment]
@dkhabarov paste somewhere ur playbook and other files for testing, please.

**Comment by robynbergeron** [comment]
[module: cloud/lxc/lxc_container.py]

**Comment by robynbergeron** [comment]
@dkhabarov -- Thanks for submitting this bug report, and thanks to @machacekondra @mightydok for your comments and help thus far :)

@cloudnull -- thanks in advance for taking a look at this bug report and for your ongoing work in maintaining this module.

**Comment by robynbergeron** [comment]
@dkhabarov -- if you could paste the information requested by @mightydok for assistance in troubleshooting, that would be helpful.

**Comment by mightydok** [comment]
@robynbergeron same error as @dkhabarov posted, fix #639 dont help.
Playbook:

```
---
- hosts: all
  become: true

  vars: 
    containers:
        - {name: 'webrails1', network_ipv4_gateway: 10.0.3.1, network_ipv4_address: 10.0.3.2, network_ipv4_broadcast: 10.0.3.255, state: 'started' }
    is_container_host: True

  tasks:
  - name: manage containers
    lxc_container:
      name: "{{ item.name }}"
      template: "{{ ansible_distribution | lower }}"
      state: "{{ item.state | default('started') }}"
      container_config:
        - "lxc.start.auto = 1"
        - "lxc.network.ipv4 = {{ item.network_ipv4_address }}"
        - "lxc.network.ipv4.gateway = {{ item.network_ipv4_gateway }}"
        - "lxc.mount.entry = /data data/ none rw,bind,uid=33,gid=33 0 0"
      template_options: "--release {{ ansible_distribution_release }}"
    with_items: containers
    when: is_container_host is defined and item.state == 'started'
    tags: configure_lxc
```

**Comment by mightydok** [comment]
@robynbergeron this bug was already fixed: https://github.com/ansible/ansible-modules-extras/commit/24cfcd2497f1d68cfffaef6f5afa355018f663c2 + #639 bug fix and module works with @dkhabarov config.

Ubuntu have very old modules, so they need updates.
